zopkio/reporters/junit_reporter.py

```diff
@@ -0,0 +1,112 @@
+# Copyright 2014 LinkedIn Corp.
```

Comment:
It was copied from html_reporter file. I dont think i change the year there.

Chuong
On Sep 15, 2015 1:34 PM, "Joshua Ehrlich" <notifications@github.com> wrote:

> In zopkio/reporters/junit_reporter.py
> <https://github.com/linkedin/Zopkio/pull/90#discussion_r39561126>:
>
> > @@ -0,0 +1,112 @@
> > +# Copyright 2014 LinkedIn Corp.
>
> change year? Awesome
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/linkedin/Zopkio/pull/90/files#r39561126>.
>